packages/datadog-plugin-kafkajs/src/index.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+function createWrapProducer (tracer, config) {
+  return function wrapProducer (createProducer) {
+    return function producerWithTrace () {
+      const serviceName = config.service || `${tracer._service}-kafka`
+      const createdProducer = createProducer.apply(this, arguments)
+
+      const send = createdProducer.send
+
+      const tags = {
+        'service.name': serviceName,
+        'span.kind': 'producer',
+        'span.type': 'queue',
+        component: 'kafka'
+      }
+
+      createdProducer.send = tracer.wrap('kafka.producer.send', { tags }, function (...args) {
```

Comment:
The span name should be in the form `<lib>.<action>` so for example `kafka.produce` or `kafka.send`, or basically whatever makes the most sense depending on Kafka's own terminology.